java/src/main/java/ai/rapids/cudf/PinnedMemoryPool.java

```diff
@@ -128,9 +128,9 @@ public static synchronized void initialize(long poolSize, int gpuId) {
    *
    * @param poolSize size of the pool to initialize.
    * @param gpuId    gpu id to set to get memory pool from, -1 means to use default
-   * @param setCuioHostMemoryResource true if this pinned pool should be used by cuIO for host memory
+   * @param setCudfPinnedPoolMemoryResource true if this pinned pool should be used by cuIO for host memory
```

Comment:
```suggestion
   * @param setCudfPinnedPoolMemoryResource true if this pinned pool should be used by cuDF for host memory
```

Reply:
Also:
```suggestion
   * @param setCudfPinnedPoolMR true if...
```

Reply:
I did not get your comment @ttnghia, could you explain?